conda_smithy/templates/conda-forge.yml.skel.tmpl

```diff
@@ -0,0 +1,11 @@
+recipe_dir: {{ recipe_directory }}
+skip_render:
+  - README.md
+  - LICENSE.txt
+  - .gitattributes
+  - .gitignore
+  - build-locally.py
+  - LICENSE
+  - CONTRIBUTING.md
+  - ISSUE_TEMPLATE.md
+  - PULL_REQUEST_TEMPLATE.md
```

Comment:
These three files are not automatically generated anymore.

Reply:
Right, but they are automatically removed, which I want to prevent

Reply:
Got it. Thanks for the info.

Should this be `.github/CONTRIBUTING.md` ?

Reply:
Is there a way to do this in a cross-platform way?  I chose to skip on basenames because they are unique